internal/pkg/db/db_test.go

```diff
@@ -38,6 +39,8 @@ type dbSuite struct {
 	documents []*sbom.Document
 }
 
+var documentTags = [][]string{{"tag1", "tag2"}, {"tag2", "tag3"}}
+
 func (dbs *dbSuite) SetupSuite() {
 	backend, err := db.NewBackend(db.WithDatabaseFile(db.DatabaseFile))
```

Comment:
Test using in-memory database if possible

```suggestion
	backend, err := db.NewBackend(db.WithDatabaseFile(":memory:"))
```